packages/components/src/components/Menu/index.tsx

```diff
@@ -194,6 +194,7 @@ const MenuButton = props => (
         width: 'auto',
         // disable scale feedback of buttons in menu
         // to make the menu feel less jumpy
+        transform: 'scale(1)',
```

Comment:
We use `Button` for `Menu.Button` which has a scale animation on click, this looks very jumpy in the case of Menu because you get additional visual feedback with the Menu opening